coffeecard/CoffeeCard.WebApi/Controllers/v2/AccountController.cs

```diff
@@ -0,0 +1,52 @@
+using System.Threading.Tasks;
+using CoffeeCard.Common.Errors;
+using CoffeeCard.Library.Services;
+using CoffeeCard.Library.Utils;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CoffeeCard.WebApi.Controllers.v2
+{
+    /// <summary>
+    /// Controller for creating and managing user accounts
+    /// </summary>
+    [ApiController]
+    [ApiVersion("2")]
+    [Route("api/v{version:apiVersion}/account")]
+    [Authorize]
+    public class AccountController : ControllerBase
+    {
+        private readonly IAccountService _accountService;
+        private readonly ClaimsUtilities _claimsUtilities;
+        
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AccountController"/> class.
+        /// </summary>
+        public AccountController(IAccountService accountService, ClaimsUtilities claimsUtilities)
+        {
+            _accountService = accountService;
+            _claimsUtilities = claimsUtilities;
+        }
+
+        /// <summary>
+        /// Delete an account
+        /// </summary>
+        /// <param name="email">email of account to delete</param>
+        /// <response code="204">Successful account deletion</response>
```

Comment:
The account is not deleted when 204 is returned. We should to the documentation in the summary section that an anonymization email is sent etc and the 204 response represent a successful initiation of the anonymization request